device/bgp/bgpv6.yaml

```diff
@@ -1,23 +1,39 @@
 components:
   schemas:
-    Device.Bgpv6:
+    Bgp.V6peer:
```

Comment:
V6peer vs. V6Peer?  (other similar situations had it capitalized)

Reply:
Right, 'V6Peer'.